rancher/charts
# [dev-v2.9] Bump shell image used by rancher-monitoring

## Issue: 


## Problem

## Solution


## Testing

To test:

1. Rancher Monitoring of an older version needs to be installed,
2. Update Rancher Monitoring to version: `104.0.0-rc1+up57.0.3`
3. Verify that the upgrade job it creates uses shell version: `v0.2.1-rc.7`
4. Verify that the upgrade job completes to a success.


## Engineering Testing
### Manual Testing


### Automated Testing


## QA Testing Considerations


### Regressions Considerations


## Backporting considerations
